nvflare/fuel/f3/cellnet/utils.py

```diff
@@ -93,3 +99,39 @@ def decode_payload(message: Message, encoding_key=MessageHeaderKey.PAYLOAD_ENCOD
         # assume to be bytes
         pass
     message.remove_header(encoding_key)
+
+
+def format_size(size, binary=False):
```

Comment:
I thought "binary" should be True by default. 
But I wonder whether we need this option. What's the use case?

Reply:
To conform with convention and standard, when we say MB, GB, it's non binary mode. That's what we have been using so left it as default.